pkgs/top-level/all-packages.nix

```diff
@@ -27007,6 +27007,9 @@ with pkgs;
     pkgs' = pkgs;  # default pkgs used for bootstrapping the emacs package set
   };
 
+  # This alias should live in aliases.nix but that would cause Hydra not to evaluate/build the packages.
```

Comment:
Without this, everyone would be forced to use `emacs.pkgs`?

Reply:
No, `emacsPackages` is moved because Hydra evaluates nixpkgs with `allowAliases = false` which currently means they're not evaluated at all.
The default value for `allowAliases` is `true` so almost all users could use `emacsPackages` instead of `emacs.pkgs`, it just wasn't permitted within nixpkgs.

The better solution to this is probably to have a completely separate Hydra jobset for Emacs packages and move `emacsPackages` back to aliases.nix.